openapi_python_client/parser/properties/property.py

```diff
@@ -69,17 +82,25 @@ def get_imports(self, *, prefix: str) -> Set[str]:
             imports.add(f"from {prefix}types import UNSET, Unset")
         return imports
 
-    def to_string(self) -> str:
-        """ How this should be declared in a dataclass """
+    def to_string(self, query_parameter: bool = False) -> str:
+        """
+        How this should be declared in a dataclass
+
+        Args:
+            query_parameter: True if the property's type is being used for a query parameter
+        """
         default: Optional[str]
         if self.default is not None:
             default = self.default
         elif not self.required:
-            default = "UNSET"
+            if query_parameter:
+                default = "None"
+            else:
+                default = "UNSET"
```

Comment:
I don't think this change is needed (since they have the same outcome) and it just adds extra complexity.

```suggestion
            default = "UNSET"
```